modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,32 @@
+// Module filename: nw-customize-ingress-error-pages.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-customize-ingress-error-pages_{context}"]
+= Customizing Ingress Controller error pages
+
+As a cluster administrator, you can use the `HttpErrorCodePages` field to specify custom error code pages for the Ingress Controller. By default, OpenShift router serves a 503 error page when the server is not running or when the route is incorrect or non-existent. When you specify a custom error code page, {product-title} serves either a 503 or 404 error page is returned when the requested URL does not exist, instead of returning the default HTTP 503 error page.
+
+.Procedure
+. Create a configmap named "my-custom-error-code-pages" in the openshift-config namespace.
++
+[source,terminal]
+----
+$ oc -n openshift-config create configmap my-custom-error-code-pages
+--from-file=error-page-503.http
+--from-file=error-page-404.http
+----
++
+[NOTE]
+====
+The configmap keys have two available formats, error-page-503.http and error-page-404.http.
+====
+
+. Patch the ingresscontroller to reference the `my-custom-error-code-pages` config map by name.
++
+[source,terminal]
+----
+$ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"HttpErrorCodePages": "my-custom-error-code-pages"}}' --type=merge
```

Comment:
H is small h in HttpErrorCodePages   oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge --patch='{"spec":{"httpErrorCodePages":{"name":"my-custom-error-code-pages"}}}'